docs/src/main/paradox/index.md

```diff
@@ -4,7 +4,12 @@ This is a multi-part guide to walk you through how to make a RESTful API with JS
 
 We’ll demonstrate with a “best practices” REST API.  You can get source code for this guide two ways:
 
-* [Pre-packaged bundle](https://example.lightbend.com/v1/download/play-rest-api)
+* Download a pre-packaged bundle with this link [https://example.lightbend.com/v1/download/play-rest-api](https://example.lightbend.com/v1/download/play-rest-api)
+    ```bash
```

Comment:
triple quotes must not have leading whitespace and should have paragraph breaks.